eapim/policy.xml

```diff
@@ -0,0 +1,34 @@
+<policies>
+
+    <inbound>
+
+        <base />
+
+        <validate-jwt header-name="Authorization" failed-validation-httpcode="401" failed-validation-error-message="Unauthorized. Access token is missing or invalid.">
+            <openid-config url="{{oauth-eapim-config-url}}" />
+            <required-claims>
+                <claim name="aud">
+                    <value>{{oauth-spi-aud-claim}}</value>
+                </claim>
+            </required-claims>
+        </validate-jwt>
+
+        <set-header name="X-Functions-Key" exists-action="skip">
+            <value>{{squasher-function-app-key}}</value>
```

Comment:
You'll want to update this to be `eventbroker-function-app-key`.

Reply:
Darn; copy/paste failed me again!